src/main/java/org/traccar/protocol/Gl200TextProtocolDecoder.java

```diff
@@ -1033,36 +1033,33 @@ private Object decodeEri(Channel channel, SocketAddress remoteAddress, String[]
         return positions;
     }
 
-    private static final Pattern PATTERN_IGN = new PatternBuilder()
-            .text("+").expression("(?:RESP|BUFF):GT[IV]G[NF],")
-            .expression("(?:.{6}|.{10})?,")      // protocol version
-            .number("(d{15}|x{14}),")            // imei
-            .expression("[^,]*,")                // device name
-            .number("d+,")                       // ignition off duration
-            .expression(PATTERN_LOCATION.pattern())
-            .number("(d{5}:dd:dd)?,")            // hour meter
-            .number("(d{1,7}.d)?,")              // odometer
-            .number("(dddd)(dd)(dd)")            // date (yyyymmdd)
-            .number("(dd)(dd)(dd)").optional(2)  // time (hhmmss)
-            .text(",")
-            .number("(xxxx)")                    // count number
-            .text("$").optional()
-            .compile();
-
-    private Object decodeIgn(Channel channel, SocketAddress remoteAddress, String sentence) {
-        Parser parser = new Parser(PATTERN_IGN, sentence);
-        Position position = initPosition(parser, channel, remoteAddress);
-        if (position == null) {
+    private Object decodeIgn(
```

Comment:
Please also don't forget to attach the documentation for future reference.

Reply:
documentation attached